sparkle/src/components/NewDropdown.tsx

```diff
@@ -339,6 +340,58 @@ const NewDropdownMenuShortcut = ({
 };
 NewDropdownMenuShortcut.displayName = "NewDropdownMenuShortcut";
 
+interface NewDropdownMenuSearchbarProps {
```

Comment:
IMHO would be clearer written with an `extends SearchbarProps` + `Omit` if this makes sense, wdyt?